### PR TITLE
Added quality checks to queue script template

### LINKED ISSUE #1345: check for empty arrays in output
place check in queue script, question is - what to do empty arrays are detect - @jasonfleming?

### PR DESCRIPTION
**Pull request by jasonfleming**
resolves #1345 ... testing now

### CODE REVIEW

File: qscript.template [hunk 1]
```diff
@@ -128,23 +142,135 @@ for layer in ${layers[@]}; do
...
-                  echo "$THIS: No numerical instability detected in '$file' after completion of $jobtype.$layer job ${%JOBID%}." 2>&1 | awk -v level=INFO -v this=$THIS -f $SCRIPTDIR/monitoring/timestamp.awk | tee --append $SCENARIOLOG
+                  echo "cycle $CYCLE: $SCENARIO: No numerical instability detected in '$file' after completion of '$jobtype.$layer' job '${%JOBID%}'." 2>&1 | awk -v level=INFO -v this=$THIS -f $SCRIPTDIR/monitoring/timestamp.awk | tee --append $SCENARIOLOG
                else
                   ERROVALUE=1
-                  ERROMSG="$ERROMSG Detected $numMsg numerical instability messages in '$file' after completion of $jobtype.$layer job ${%JOBID%}."
+                  ERROMSG="$ERROMSG Detected '$numMsg' numerical instability messages in '$file' after completion of '$jobtype.$layer' job '${%JOBID%}'. "
                   RUNSUFFIX="error"
                fi
             fi
          done
```
Comment (wwlwpd) [inline qscript.template:152]: This is awesome. Would it be too much to as to put this check into a seperate script? 2 reasons:

1. this is obviously generally useful (and I'd like to use it with `asgs-mon` in some way)
2. at some point I'd really like to redo the queue scripts so they use Template::Toolkit

Comment (jasonfleming) [reply qscript.template:152]: Yes I think so, I will have a look to see how hard this might be.

**Comment by jasonfleming** [comment]
@wwlwpd I have successfully moved these quality checks to a separate script and tested on `debian`. Now testing on SuperMIC. If successful, this will be ready to merge.

File: qscript.template (hunk 1 above)
Comment (jasonfleming) [reply qscript.template:140]: I've removed this from the queue script template and placed in a separate script in `util/output`.

**Comment by jasonfleming** [comment]
@wwlwpd I've now tested and confirmed that this is working, please review and merge if acceptable. :-)

**Review by wwlwpd — APPROVED** [review]
Looks great. Nice to finally have this kind of thing.

**Pull request merged**